src/utils/store/store-initializer.ts

```diff
@@ -16,11 +21,27 @@ type CreateStoreInitializerArgs<T extends object> = {
   enhancers?: Array<StoreEnhancer>;
 };
 
+/**
+ * Util that used for creating the application's initStore.
```

Comment:
```suggestion
 * Utility that creates a function generating a store for an application.
```

Reply:
@ipakhomov should we change this text in README too?